src/nyc_trees/apps/survey/tests.py

```diff
@@ -261,3 +263,68 @@ def _assert_blockface_achievements(self, amount_blockfaces,
         self.assertEqual(amount_achievements, len(achievements))
         achievements = get_achievements_for_user(self.other_user)['achieved']
         self.assertEqual(amount_achievements, len(achievements))
+
+
+class TilerTests(SurveyTestCase):
+    def _exec_tiler_query(self, filename, **kwargs):
+        group_id = kwargs.get('group_id', '0')
+
+        # The path to the tiler SQL queries is defined as a synced
+        # folder in the Vagrant file.
+        path = os.path.join('/opt/tiler/sql/', filename)
```

Comment:
It would be more conventional to have this path saved in ``settings/test.py`` and ``settings/development.py.``. It behaves sort of like the static assets directory.